### PR TITLE
refactor/1608 - Add Japanese Airline Files

### LINKED ISSUE #1608: Add/Update Japanese Airlines
Update and add Japanese airlines for the development of Japanese airports.

### PR DESCRIPTION
**Pull request by tomomasuda**
Resolves #1608 

The purpose of this pull request is to add or update the following 9 airlines in Japan.
(Air Asia Japan, Air Do, Air Peach, All Nippon Airways, Fuji Dream Airlines, IBEX Airlines, Japan Airlines, Jetstar Japan, Star Flyer, Skymark)

### CODE REVIEW

File: assets/airlines/ana.json [hunk 1]
```diff
@@ -4,54 +4,54 @@
...
-            ["A320", 2],
+            ["A320", 4],
             ["A321", 4],
             ["B737", 8],
-            ["B738", 39]
-        ],
-        "b788": [
-            ["B788", 36]
+            ["B738", 39],
+            ["B763", 33],
```
Comment (Fechulo) [inline assets/airlines/ana.json:42]: Aircraft like the 777s and 787s should be in the long fleet not in the short

Comment (tomomasuda) [reply assets/airlines/ana.json:42]: As for the 777-200 and 777-300s, ANA uses them only for domestic flights (i.e. short-haul flights — 1-3 hours) and only uses the 77W for the long haul.

For the other short-haul category aircraft, I've adjusted to the number of short-haul specifications.

Comment (Fechulo) [reply assets/airlines/ana.json]: Let me rephrase. While I don't really have a problem with 777s and 787s in the short section, they should still be present in the long haul section (the 787s and 77Ws anyway), as many airports that include ANA in their airport files would rely on those aircraft being part of the long fleet. It might be worth adding a 'medium' fleet for the heavy aircraft used for those 1-3 hour long flights you mention.

Comment (tomomasuda) [reply assets/airlines/ana.json]: Okay! Will do. Please give me a moment to push that before reviewing it again. Thank you!

**Comment by Fechulo** [comment]
@tomomasuda Take a look at the other comments I've left:

> Make sure you don't delete fleets that are being used in other airport files. For example, the jfk fleet in JAL.json is used in KJFK, its absence will cause the sim to crash.

> Fair enough, let me rephrase. While I don't really have a problem with 777s and 787s in the short section, they should still be present in the long haul section (the 787s and 77Ws anyway), as many airports that include ANA in their airport files would rely on those aircraft being part of the long fleet. It might be worth adding a 'medium' fleet for the heavy aircraft used for those 1-3 hour long flights you mention.

File: assets/airlines/ana.json (hunk 1 above)
Comment (tomomasuda) [reply assets/airlines/ana.json]: > @tomomasuda Make sure you don't delete fleets that are being used in other airport files. For example, the jfk fleet in JAL.json is used in KJFK, its absence will cause the sim to crash.

Please correct me if I am wrong, but, as far as I have checked, the only JAL fleet used in KJFK is "jal/long" and same goes for the other airports

Comment (tomomasuda) [reply assets/airlines/ana.json]: For the ANA fleets, what I did was (and my original point was; I apologize for my bad communication) only put the B777-300ER in the "ana/long" as the airports including ANA at the moment is only European and American airports, which ANA only uses the B77W to operate, thus it would be absolutely unrealistic to include any other aircrafts in the category. Therefore, I have created a medium category which the short- to medium-haul flights (i.e East + Southeast Asia routes) can use.

Comment (Fechulo) [reply assets/airlines/ana.json]: > Please correct me if I am wrong, but, as far as I have checked, the only JAL fleet used in KJFK is "jal/long" and same goes for the other airports

Looks like you're right, sorry about that. I'm confused by it, but 🤷‍♂️ 

> Therefore, I have created a medium category which the short- to medium-haul flights (i.e East + Southeast Asia routes) can use

Works for me 👍

**Comment by Fechulo** [comment]
Make sure to add a changelog entry then it's ready to go✔️
